package.json

```diff
@@ -84,8 +85,7 @@
   "husky": {
     "hooks": {
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
-      "pre-push": "yarn build-tests-changed && yarn split-e2e-tests",
-      "pre-commit": "./scripts/package-json-check.sh && yarn verify-commit"
```

Comment:
`verify-commit` must not be removed

Reply:
and it's not, it's moved to the `.husky/pre-commit`